### ISSUE TITLE
Randomisation of options in survey questions

### ISSUE THREAD

**Issue by djmoffat** [BrechtDeMan/WebAudioEvaluationTool#225]
Feature Request:

I would like to randomise the order of options in a survey question. 

This would involve creating a new `<surveyentry>` attribute of `randomise`, and return the correct answer.
Someone might also want it to return the presentation order?

**Comment by djmoffat** [comment]
Just to note, I am not particularly interested in the presentation order, but it guess it is important to record in some way.

**Comment by nickjillings** [comment]
Implemented in 281de46

**Comment by nickjillings** [comment]
```xml
<surveycheckbox id="checkboxtest" randomise="true">
    ....
</surveycheckbox>
```
